### ISSUE TITLE
--browser flag not exist

### ISSUE THREAD

**Issue by noririco** [ngChile/ngchile-packages#11]
Hi,
I want to Lunch by Path
cypress run --browser /usr/bin/chromium
or
cypress open --browser /usr/bin/chromium

[https://github.com/ngChile/ngchile-packages/blob/master/packages/cypress-cucumber-schematics/src/builders/cypress/schema.json](Here) you can see you don't provide this option

**Comment by ashblox** [comment]
I'm actually running into this same issue. I'm testing file download and I cannot set the default download directory in Electron ([see Github issue](https://github.com/cypress-io/cypress/issues/4675#issuecomment-656512552)). I'd really hate to switch builders, because this one works so well. It would be great if you could add support to specify a browser!

**Comment by gpincheiraa** [comment]
Hi guys! Sorry for the late.
I will update the executing adding all the options available right now in https://docs.cypress.io/guides/guides/command-line.html#Options

**Comment by noririco** [comment]
> Hi guys! Sorry for the late.
> I will update the executing adding all the options available right now in https://docs.cypress.io/guides/guides/command-line.html#Options

Thank you very much @gpincheiraa, Also you might want to update cypress to latest version ?
https://github.com/cypress-io/cypress/releases/tag/v5.5.0

**Comment by ashblox** [comment]
Thanks for putting this together so quickly! I tried pulling down your changes and running with an `npm link`. I'm not sure that the "headless" (or "headed") flags are working as expected, though 😕 

My angular.json looks like this:
```
"architect": {
                "e2e": {
                    "builder": "@ngchile/cypress-cucumber-schematics:cypress",
                    "options": {
                        "mode": "browser",
                        "devServerTarget": "trade-queue-app:serve"
                    },
                    "configurations": {
                        ...
                        "local-local": {
                            "mode": "console",
                            "headless": true,
                            "browser": "chrome",
                            "devServerTarget": "",
                            "env": { "myServiceUrl": "https://localhost:5132" }
                        },
```

And when I run `npm run e2e:local-local` it _does_ pick up chrome but it doesn't run in headless mode.

![image](https://user-images.githubusercontent.com/47283036/97729907-8d754480-1aa9-11eb-98f0-8a2449e7098f.png)

I have the latest version of Cypress (v. 5.5.0) and if I run `npx cypress run --headless --browser chrome` I see the expected behavior: 

![image](https://user-images.githubusercontent.com/47283036/97730086-c7dee180-1aa9-11eb-8b6a-7131e333d5e6.png)

Any ideas? The code above is from a repo that I can't share, but if you need me to put together an example repo, I'd be happy to. Let me know.

**Comment by noririco** [comment]
@ashblox  maybe its due to cypress version "cypress": "^5.0.0" ?

**Comment by gpincheiraa** [comment]
@ashblox Hi!
I add a logs for check in more detail why you cannot run headless mode. I have been testing your example and for me works.
<img width="655" alt="Captura de Pantalla 2020-10-31 a la(s) 12 40 17" src="https://user-images.githubusercontent.com/5608336/97783490-f1b20a00-1b76-11eb-9856-6343a24ea74b.png">
<img width="1354" alt="Captura de Pantalla 2020-10-31 a la(s) 12 39 51" src="https://user-images.githubusercontent.com/5608336/97783491-f37bcd80-1b76-11eb-9c84-cf0f691ce89b.png">

**Comment by gpincheiraa** [comment]
> > Hi guys! Sorry for the late.
> > I will update the executing adding all the options available right now in https://docs.cypress.io/guides/guides/command-line.html#Options
> 
> Thank you very much @gpincheiraa, Also you might want to update cypress to latest version ?
> https://github.com/cypress-io/cypress/releases/tag/v5.5.0

It is not mandatory put explicitly the `5.5.0` version because using `^5.0.0` ensures that the last minor version will be installed. in this case the `v5.5.0`. Reference: https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept

I will not chage the `^5.0.0` because I want give more compatibility with people that does not use the last version

**Comment by gpincheiraa** [comment]
@noririco @ashblox : I closed this issue in the release `1.0.0-rc.8`. Please check if works for your projects and we can re-open this issue if any have some problem with this version or open a new one.
Thank you for your feedback!

**Comment by ashblox** [comment]
Thanks @gpincheiraa ! It does seem to be working now. Not sure what I had wrong the first time.
![image](https://user-images.githubusercontent.com/47283036/97887685-3ca75b00-1cf8-11eb-91c4-4b20ea551fa0.png)

Much appreciate your help with this 🤝

**Comment by gpincheiraa** [comment]
@ashblox  I just released the version `1.1.0`. Can you try to use and check if everything seems ok?

**Comment by ashblox** [comment]
Yup, just ran the same command with no issues! The help text is useful too. Thanks again!
